### PR TITLE
support completions for command aliases

### PR DESCRIPTION
**Pull request by rajatjindal**
This is the second part of PR #588 

this will fix https://github.com/kubernetes/helm/issues/3171

### CODE REVIEW

**Comment by rajatjindal** [comment]
requesting review from @eparis

**Comment by rajatjindal** [comment]
ping @eparis

**Comment by rajatjindal** [comment]
Ping @eparis

File: bash_completions.go [hunk 1]
```diff
@@ -443,6 +453,14 @@ func writeRequiredNouns(buf *bytes.Buffer, cmd *Command) {
 	}
 }
 
+func writeCmdAliases(buf *bytes.Buffer, cmd *Command) {
+	sort.Sort(sort.StringSlice(cmd.Aliases))
+	for _, value := range cmd.Aliases {
+		buf.WriteString(fmt.Sprintf("    command_aliases+=(%q)\n", value))
+		buf.WriteString(fmt.Sprintf("    aliashash[%q]=%q\n", value, cmd.Name()))
```
Comment (eparis) [inline bash_completions.go]: Does this not also need a BASH_VERSION check? If not, this PR LGTM....

**Comment by rajatjindal** [comment]
I added the check to be on safer side, kindly review again

**Comment by rajatjindal** [comment]
Hi eparis

if you do merge this, do you think we can also tag a new release for cobra when you do that?

**Comment by rajatjindal** [comment]
Ping @eparis

**Comment by rajatjindal** [comment]
Ping @eparis

**Comment by eparis** [comment]
I'll let it sit for a couple days and if no one complains about anything tag. That sound ok?

**Comment by rajatjindal** [comment]
Sure, sounds good, thanks

**Pull request merged**